projects/components/src/spotlight-search/spotlight-search.component.html

```diff
@@ -0,0 +1,47 @@
+<clr-modal
+    [(clrModalOpen)]="open"
+    [clrModalStaticBackdrop]="false"
+    (keydown.arrowup)="onArrowUp($event)"
+    (keydown.arrowdown)="onArrowDown($event)"
+    (keydown.enter)="onEnterKey($event)"
+>
+    <div class="modal-body">
+        <div class="search-input-container">
+            <clr-icon shape="search" size="20"></clr-icon>
+            <input
+                #searchInput
+                type="text"
+                class="clr-input"
+                [placeholder]="placeholder || ''"
+                [(ngModel)]="searchCriteria"
+            />
+        </div>
+        <div class="search-result-container">
+            <div *ngFor="let searchSection of searchSections" class="search-result-section">
+                <div
+                    *ngIf="
+                        searchSections.length > 1 &&
```

Comment:
Please avoid long expressions in templates https://angular.io/guide/template-syntax#simplicity Create a simple method